### PR TITLE
Add observability setup to Kubernetes install guide

### PR DESCRIPTION
**Pull request by david-martin**
Based on [readme](https://github.com/Kuadrant/kuadrant-operator/tree/main/config/observability#observability-stack-guide).

Just for Istio (not Envoy Gateway) as there is extensive use of istio specific metrics.

### CODE REVIEW

**Comment by R-Lawton** [comment]
👀

File: doc/install/install-kubernetes.md [hunk 1]
```diff
@@ -113,6 +113,37 @@ kubectl wait --for=jsonpath='{.status.state}'=AtLatestKnown subscription/my-kuad
 
 Kuadrant is now ready to use.
 
+### (Optional) Observability setup (Istio only)
```
Comment (R-Lawton) [inline doc/install/install-kubernetes.md:116]: The formatting isnt showing properly

Comment (david-martin) [reply doc/install/install-kubernetes.md]: fixed. Odd space character after hash

**Review by R-Lawton — APPROVED** [review]
/lgtm

**Pull request merged**